bothan-binance/src/worker.rs

```diff
@@ -39,6 +40,7 @@ impl AssetWorker for Worker {
         opts: Self::Opts,
         store: &S,
         ids: Vec<String>,
+        metrics: &Metrics,
```

Comment:
I'd probably be better for the metric itself to be build per source rather than be passed into it (for all workers)